peekingduck/pipeline/nodes/model/master_map.yml

```diff
@@ -4,6 +4,7 @@
 efficientdet: map_effdet
 yolo: map_yolo
 yolox: map_yolo
+mask_rcnn: map_effdet
```

Comment:
Does this really use efficient det's mapping? The `num_classes` in efficient det's config file is 90 while in mask rcnn's config file, it's 91

Reply:
The Mask-RCNN by default uses 91 class, one of the classes is the background which is not applicable during inference. I have deducted the default predicted labels by 1 in `detector.py` (e.g. person class id 1 --> 0) and the resulting label ids are the same as the efficientdet's mapping